index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: lesson
+root: .
+lastupdated: April 16, 2015
+contributors: ["Ethan White","Greg Wilson","Josh Herr","Sophie Clayton","Tracy Teal"]
+maintainers: []
+domain: Ecology
+topic: SQL
+software: SQL
+dataurl: http://dx.doi.org/10.6084/m9.figshare.1314459
+status: Teaching
+---
+
+<!-- USING THIS LESSON TEMPLATE -->
+<!-- Lesson specific information is taken from the YAML header at the top of the page -->
+
+<!-- THE LESSON INFORMATION -->
+
+<!-- Get the information from _data/info.yml -->
+
+#Data Carpentry {{ page.topic }} for {{ page.domain }}
```

Comment:
I'd suggest we be consistent and put a space after the heading-level defining `#`s.